daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Time.scala

```diff
@@ -17,25 +16,28 @@ import scala.util.Try
 
 object Time {
 
-  case class Date private (days: Int) extends Ordered[Date] {
+  class Date private (val days: Int) extends Ordered[Date] {
```

Comment:
move as normal class, as

1. toString is already implemented
2. builder is private 
3. copy is insecure
4. pattern matching not very useful, as contains only one field

We have to add equals thought.